.github/workflows/summary_check.yaml

```diff
@@ -0,0 +1,20 @@
+name: summary check
+# This workflow is triggered on check runs
+on: 
+  status
+
+jobs:
+  summary_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: summary_check
+        uses: sofisl/checkRunAction@v10
+        with:
+          conclusion: ${{ github.event.status.state }}
+          pull_request_head_sha: ${{ github.event.status.sha }}
+          author: ${{ github.actor }}
+          myToken: ${{ github.token }}
```

Comment:
I think this should be `secrets.GITHUB_TOKEN` then it's magically populated for you.